benchmarks/maxtest/Getting_Started.md

```diff
@@ -0,0 +1,88 @@
+### Getting starting with using the maxtest tool
+
+*Note: This is currently in preview stage and only supported for v5p and v6e*
+
+The main intent of this tool is to quickly verify whether the provisioned TPU
+nodepool can run a Maxtext workload **without having to
+use XPK**. This is useful for debugging provisioned nodepools for potential hardware issues.
```

Comment:
What's wrong with using XPK for this? Also, the `benchmark_runner` generally uses XPK under the hood. Is it not using that here?